pivot-growth.jl

```diff
@@ -146,10 +151,51 @@ function run_model(n)
     val,B,MOI.FEASIBLE_POINT
 end
 
+"""
+    run_model_new(n)
+
+Maximize pivot growth for real matrix with complete pivoting.
+"""
+function run_model_new(n)
+    start_matrix = randn(n, n)
+    thestart = genp(geperm(start_matrix))
+    thestart = genp(thestart[:,:,1] * Diagonal(sign.(thestart[k,k,k] for k = 1:n)))
+    thestart ./= thestart[1,1,1]
+    model = Model(Ipopt.Optimizer)
+    set_optimizer_attribute(model, "max_iter", 500)
+    set_silent(model)
+    indices = [(i, j, k) for k in 1:n for i in k:n for j in k:n]
+    @variable(model, x[(i,j,k) in indices], start = thestart[i,j,k])
+    for k in 1:n
+        set_lower_bound(x[(k,k,k)], 0)
+    end
```

Comment:
The main change I made was this:

`@constraint(model, x >= 0)` adds a new row to the constraint matrix. It is more efficient to set a bound.